backend/.envrc.example

```diff
@@ -0,0 +1,16 @@
+SERVER_PORT=
+OBSERVER_DB_HOST=
+OBSERVER_DB_PORT=
+OBSERVER_DB_USER=
+OBSERVER_DB_PASSWORD=
+OBSERVER_DB_NAME=
+OBSERVER_DB_SSL_MODE=
+ERPC_DB_HOST=
+ERPC_DB_PORT=
+ERPC_DB_USER=
+ERPC_DB_PASSWORD=
+ERPC_DB_NAME=
+ERPC_DB_SSL_MODE=
+GENESIS_TIMESTAMP=
+SLOT_DURATION=
+SLOTS_PER_EXPOCH=
```

Comment:
If I'm not mistaken, you must prepend `export ` before all declarations in a `.envrc` file.